views/overview/tabs.jade

```diff
@@ -0,0 +1,20 @@
+include ../mixins/overviewFields
+
+if (tabs && tabs.length)
+    div.tabs
+        div
+            ul.nav.nav-tabs.nav-tabs-overview
+                each tab, index in tabs
+                    li(class="#{index === 0 ? 'active' : ''}")
+                        a(href='#' + tab.label + index, role='tab' data-toggle='tab')
+                            span=tab.label
+        div.tab-content
+            each tab, index in tabs
+                if (tab.fields)
+                    div(id=tab.label + index, class="#{index === 0 ? 'tab-pane active' : 'tab-pane'}")
```

Comment:
Should move this above the if statement on `L13` and you can remove the duplicates from the rest of the `if` statements since you need to output that markup regardless of the `if` statements.